Cargo.toml

```diff
@@ -16,6 +16,7 @@ no-default-features = true
 
 [dependencies]
 rustls = { version = "0.23", default-features = false, features = ["std"] }
+time = { version = "0.3.43", default-features = false, optional = true }
```

Comment:
Please add the feature under `[features]` section:
`time = ["dep:time"]`

Reply:
it's is implicit,`optional = true` already creates a feature named `time` (indeed I used it in the code)

Reply:
Anyway added to be explicit